modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/LoaderError.scala

```diff
@@ -43,7 +43,13 @@ object LoaderError {
   }
 
   /** Error happened during DDL-statements execution. Critical */
-  final case class MigrationError(message: String) extends LoaderError
+  sealed trait MigrationError extends LoaderError
```

Comment:
Interesting that we were never using `MigrationError` before your PR!  I guess it was left over from an earlier version of RDB Loader.

Can we delete this trait now?  Do we really need `TableCommentError` to extend `MigrationError`?

Reply:
We are not making use of this nested error hierarchy, we can delete that for sure.